src/PostgREST/SchemaCache.hs

```diff
@@ -442,7 +452,6 @@ funcsSqlQuery pgVer = [q|
   LEFT JOIN pg_class comp ON comp.oid = t.typrelid
   LEFT JOIN pg_description as d ON d.objoid = p.oid
   LEFT JOIN LATERAL unnest(proconfig) iso_config ON iso_config like 'default_transaction_isolation%'
-  LEFT JOIN LATERAL unnest(proconfig) timeout_config ON timeout_config like 'statement_timeout%'
```

Comment:
Use this query here:

```sql
  LEFT JOIN LATERAL (
    SELECT
      coalesce(array_agg(row(
        substr(setting, 1, strpos(setting, '=') - 1),
        lower(substr(setting, strpos(setting, '=') + 1))
      )), '{}') as kvs
    FROM unnest(proconfig) setting
    WHERE setting not like 'default_transaction_isolation%'
  ) func_settings ON TRUE
```

The decoding would then need to use `compositeArrayColumn`, like done here:

https://github.com/PostgREST/postgrest/blob/49584728b6423131712518e2f0803347c80069a1/src/PostgREST/Config/Database.hs#L181-L182